src/components/TimePicker.tsx

```diff
@@ -0,0 +1,81 @@
+import './styles/time_picker.less'
+
+import React from 'react'
+import ClockIcon from './icons/ClockIcon.svg'
+
+import CustomParseFormat from 'dayjs/plugin/customParseFormat'
+import dayjs, { Dayjs } from 'dayjs'
+import Select from './select'
+import classNames from 'classnames'
+
+dayjs.extend(CustomParseFormat)
+
+const MINUTES_INTERVAL = 5
+
+type Props = {
+    minutesInterval?: number
+    onTimeChanged?: (value: string) => void
+    min?: string
+    max?: string
+    defaultValue?: string
+    className?: string
+    optionsClassName?: string
+}
+
+export default function TimePicker({
+    minutesInterval,
```

Comment:
you can set up the fallback value if absent right here when de-structuring:
```suggestion
    minutesInterval = MINUTES_INTERVAL,
```